subworkflows/nf-scil/preproc_t1/tests/nextflow_synth.config

```diff
@@ -0,0 +1,14 @@
+process {
+    publishDir = { "${params.outdir}/${task.process.tokenize(':')[-1].tokenize('_')[0].toLowerCase()}" }
+    withName: "IMAGE_RESAMPLE" {
+        ext.voxel_size = 1
+        ext.interp = "lin"
+    }
+    withName: "BETCROP_CROPVOLUME_T1" {
+        ext.output_bbox = true
+        ext.first_suffix = "t1"
+    }
+    withName: "BETCROP_SYNTHBET" {
+        memory = 8.GB
```

Comment:
I'm not sure you need the 8GB here. The runners are limited to 4, so it should run limiting RAM to 4GB. Can you try locally ?

Reply:
When running the synthbet tests, process gets killed:

`Processing frame (of 1): 1 .command.sh: line 6:    30 Killed`

Could lower it to 7.GB, but anything config lower crashes.